sageresearch-app-sdk/schemas/org.sagebionetworks.research.sageresearch.dao.room.ResearchDatabase/1.json

```diff
@@ -0,0 +1,226 @@
+{
+  "formatVersion": 1,
```

Comment:
How did you create this file?

Reply:
https://github.com/Sage-Bionetworks/BridgeAndroidSDK/pull/174/files/53f93d2b77852fe3d66a712d05c843c165958bbe#diff-d3748474ee93c6c63f4108fbc7239b58L14